repo/blob/s3/s3_storage.go

```diff
@@ -225,6 +226,14 @@ func toBandwidth(bytesPerSecond int) iothrottler.Bandwidth {
 	return iothrottler.Bandwidth(bytesPerSecond) * iothrottler.BytesPerSecond
 }
 
+func getCustomTransport(insecureSkipVerify bool) (transport *http.Transport) {
+	customTransport := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
i think &http.Transport{ TLSClientConfig: &tls.Config{InsecureSkipVerify: true } } is enough. don't think you need to clone it at all. This is slightly counter-intuitive API, but I __think__ this will work

Reply:
Thanks Jarek @jkowalski . I've updated the PR.